dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/metrics/MethodCallSamplingRate.kt

```diff
@@ -8,9 +8,12 @@ package com.datadog.android.core.metrics
 
 /**
  * Sampling rates for Method Call telemetry.
- * @param rate the rate to sample at.
+ * @param rate the rate to sample at (between 0 and 100).
  */
 enum class MethodCallSamplingRate(val rate: Float) {
+    ALL(rate = 100.0f),
+    HIGH(rate = 10.0f),
+    MEDIUM(rate = 1.0f),
     DEFAULT(rate = 0.1f),
```

Comment:
minor: I think there is a bit of naming clash between `MEDIUM` and `DEFAULT`. one can think they are synonyms.

Reply:
seems like it will be in another PR? this PR was merged without this update